charts/urban-os/Chart.yaml

```diff
@@ -5,63 +5,63 @@ name: urban-os
 version: 1.8.0
 
 dependencies:
-  - name: forklift
-    version: 3.1.1
-    repository: https://Datastillery.github.io/charts
-    condition: forklift.enabled
-  - name: odo
-    version: 1.1.0
-    repository: https://Datastillery.github.io/charts
-    condition: odo.enabled
-  - name: reaper
-    version: 1.1.0
-    repository: https://Datastillery.github.io/charts
-    condition: reaper.enabled
-  - name: valkyrie
-    version: 2.6.1
-    repository: https://Datastillery.github.io/charts
-    condition: valkyrie.enabled
   - name: andi
-    version: 2.1.0
-    repository: https://Datastillery.github.io/charts
-    condition: andi.enabled
+    version: 2.1.3
+    repository: file://../andi
```

Comment:
How are these local references going to affect the way a future developer will use the urban-os chart?

Reply:
The urbanOS chart will by default now point to the local charts in this repository so any changes in those repositories will be picked up when building the urbanos chart. This is also how we're going to turn this into a 1 PR process

Reply:
Sounds good!